src/v/redpanda/admin/api-doc/debug.json

```diff
@@ -111,6 +168,65 @@
                     "description": "Milliseconds since last update from peer"
                 }
             }
+        },
+        "self_test_report": {
+            "id": "self_test_report",
+            "description": "Self test single test report",
+            "properties": {
+                "disk_test_stoptime": {
+                    "type": "long",
+                    "description": "time to finish stub disk test"
+                },
+                "network_test_stoptime": {
+                    "type": "long",
+                    "description": "time to finish stub network test"
+                },
+                "failures": {
+                    "type": "boolean",
+                    "description": "True if any failures occurred"
```

Comment:
Would be more helpful for the user if we use a string (or vector of strings) for errors, rather than a bool (if this is true they presumably have to spelunk logs to find out why?)